content/en/Platform Deep Dive/Scans/target_auth.md

```diff
@@ -114,4 +120,61 @@ Once you have a sequence recorded, go to the target's Advanced Settings, toggle
 
 You can upload multiple sequences and enable only the one you want to use for the scan.
 
-![Login Sequence](/deepdive/scans/3.1_LoginSequence.png "Login Sequence")<br>
\ No newline at end of file
+{{% image src="/deepdive/scans/3.1_LoginSequence.png" alt="Login Sequence" %}}
+
+## API Target Authentication
+
+The DAST Scanner supports APIs with different authentication methods. You can set a fixed API key in a custom header or configure a login endpoint from which you obtain an authentication token.
```

Comment:
Perhaps we can link references to 'custom header' and 'custom parameter values' so people can quickly jump to that section from here